src/etc/man/cargo-package.1

```diff
@@ -9,7 +9,8 @@ cargo\-package \[em] Assemble the local package into a distributable tarball
 \fBcargo package\fR [\fIoptions\fR]
 .SH "DESCRIPTION"
 This command will create a distributable, compressed \fB\&.crate\fR file with the
-source code of the package in the current directory. The resulting file will
+source code of the package in the current directory. When run in a workspace,
+this command will create one \fB\&.crate\fR file per crate. The resulting file(s) will
```

Comment:
I almost feel like this isn't needed.  Its trying to describe a common pattern within cargo but incompletely